### PR TITLE
Fixups

### PR DESCRIPTION
**Pull request by blag**
* an HTML fix
* a Django template fixup
* a Python 3 fixup

### CODE REVIEW

File: tos/templates/tos/tos_check.html [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {% load i18n %}
 {% if note %}
-    <h2>{{ note }}</note>
+    <h2>{{ note }}</h2>
 {% else %}
-    <h2>{% blocktrans %}Terms of Service as of {{ tos.created|date:"SHORT_DATE_FORMAT" }}{% endblocktrans %}</h2>
```
Comment (nicholasserra) [inline tos/templates/tos/tos_check.html:5]: Was the h2 removed for a reason?

I think blocktranslate rename is only available after django 3.0. Will need to double check. If we roll with that will need to drop CI for django 2. Which is fine, as it's not longer really supported.

**Comment by nicholasserra** [comment]
Thanks! Couple questions posted. If you rebase you'll get the CI fix dropping python 3.6

**Comment by blag** [comment]
Reverted to `blocktrans` and put the `<h2>` back where it belongs.

Bonus: More conversions to f-strings (in addition to #65).

**Comment by nicholasserra** [comment]
LGTM thanks!

**Pull request merged**